src/components/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { BiRightArrowAlt } from 'react-icons/bi';
+import Country from './Country';
+import '../styles/home.css';
+
+const Home = () => {
+  const { countries, isLoading, error } = useSelector((store) => store.country);
+  const [search, setSearch] = useState('');
+  const navigate = useNavigate();
+  const filtered = countries.filter((country) => (
+    country.region.toLowerCase().includes(search.toLowerCase())));
+
+  if (isLoading) {
+    return <div>loading.......</div>;
+  }
+  if (error) {
+    return <div>Error from Api call </div>;
+  }
+
+  const handleChange = (e) => {
+    setSearch(e.target.value);
+  };
+
+  return (
+    <>
+      <div className="search-Region">
+        <input type="text" name="search" value={search} className="input-field" placeholder="search eg:Europe" onChange={(e) => handleChange(e)} />
+        <button type="button" className="btn">search</button>
+      </div>
+      <div className="container">
+        {filtered.length < 1 && <h2 className="Wrong-search">No results found</h2> }
+        {filtered.map((country) => (
+          <div
+            className="country-card"
+            key={country.cca3}
+            onClick={() => navigate(`/country/${country.cca3}`)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter') {
+                navigate(`/country/${country.cca3}`);
+              }
+            }}
+            tabIndex={0}
```

Comment:
- [ ] Please make sure to add a mic, settings, etc. icons on the header to follow the design guidelines. These should not be functional but only for design purposes.

### Your design

![image](https://github.com/clin2on3mun/explore-countries/assets/83803414/7e0fb00e-6470-4913-b999-916cf2f6f6ec)

### Given design

![image](https://github.com/clin2on3mun/explore-countries/assets/83803414/2161000a-d198-4b49-b5f9-60b132dcf108)